src/main/java/io/supertokens/inmemorydb/Start.java

```diff
@@ -2997,4 +2959,13 @@ public UserIdMapping[] getUserIdMapping_Transaction(TransactionConnection con, A
         }
     }
 
+    @Override
+    public int getUsersCountWithMoreThanOneLoginMethodOrTOTPEnabled(AppIdentifier appIdentifier) throws StorageQueryException {
+        return 0; // TODO
```

Comment:
since you have not implemented it, make sure that the feature flag test is failing for this.

Reply:
yes, the tests fail right now